.github/workflows/ios_review_pull_request.yml

```diff
@@ -26,7 +26,7 @@ jobs:
   review_pull_request:
     name: Review pull request
     runs-on: macOS-latest
-    timeout-minutes: 30
+    timeout-minutes: 60
```

Comment:
Why you made this change? 60 minutes is too much 😁

Reply:
On occasional clean builds, it can take a long time to build, with more UITest, it can take even longer. I had build failed because timeout so 60 here is for the safe side, it should never exceed 60 tho.